### PR TITLE
Bug 1281651, added sizing guidelines for etcd service nodes

### LINKED ISSUE #3447: Update etcd prereqs.
I would add the following language to the docs: 

"Please consult https://github.com/coreos/etcd/blob/master/Documentation/op-guide/hardware.md#hardware-recommendations to properly size your etcd Nodes"  

* Minimum ... 

NOTE: Openshift currently stores image meta-data in etcd,  operators should take special consideration take special care to periodically prune old images.  If you are planning to leverage a large number of images, then operators should consider placing etcd on machines with large amounts of memory and fast ssd drives.

/cc @ahardin-rh

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1281651

Also closes https://github.com/openshift/openshift-docs/issues/3447

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@timothysc @adellape PTAL

**Comment by timothysc** [comment]
lgtm, but ideally I'd like @sferich888 to once over as well.

**Review by adellape — APPROVED** [review]
LGTM

File: install_config/install/prerequisites.adoc [hunk 1]
```diff
@@ -79,7 +79,13 @@ back end; see xref:configuring-docker-storage[Configuring Docker Storage]
 below.
 
 |External etcd Nodes
-|Minimum 20 GB hard disk space for etcd data.
+a|* Minimum 20 GB hard disk space for etcd data.
+* Consult
+link:https://github.com/coreos/etcd/blob/master/Documentation/op-guide/hardware.md#hardware-recommendations[Hardware
+Recommendations] to properly size your etcd nodes.
```
Comment (sferich888) [inline install_config/install/prerequisites.adoc]: The information in this link is great, however I don't think it looks good for us to point customers at github wiki page to define such an important concept.

Comment (ahardin-rh) [inline install_config/install/prerequisites.adoc]: @timothysc thoughts?

Comment (ahardin-rh) [inline install_config/install/prerequisites.adoc]: @timothysc Please see this comment from @sferich888 . How should we move forward? Thanks!

Comment (timothysc) [inline install_config/install/prerequisites.adoc]: I don't thinks it's a concern at this time.  My 0th concern is getting the information in the hands of those who need it, and it staying current, and this is the easiest way to do that.

Comment (ahardin-rh) [inline install_config/install/prerequisites.adoc]: I'm going to merge what we have. We can continue to iterate on this as it evolves.

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../install_config/install/prerequisites.adoc#install-config-install-prerequisites[Installing a Cluster  -> Prerequisites]
|Added sizing guidelines for etcd service nodes within the xref:../install_config/install/prerequisites.adoc#hardware[Minimum Hardware Requirements table].
%

**Pull request merged**